src/Vipps.net/Infrastructure/EpaymentServiceClient.cs

```diff
@@ -9,21 +9,33 @@ namespace Vipps.net.Infrastructure
 {
     internal sealed class EpaymentServiceClient : BaseServiceClient
     {
-        internal EpaymentServiceClient(IVippsHttpClient vippsHttpClient)
-            : base(vippsHttpClient) { }
+        private readonly VippsConfigurationOptions _vippsConfigurationOptions;
+        private readonly VippsAccessTokenService _accessTokenService;
+
+        internal EpaymentServiceClient(
+            IVippsHttpClient vippsHttpClient,
+            VippsConfigurationOptions vippsConfigurationOptions,
+            VippsAccessTokenService accessTokenService
```

Comment:
Also a bit confused why the EPaymentServiceClient is injected with an AccessTokenService, but the CheckoutServiceClient is not 🤔 Probably 100% unrelated to this, but still confusing IMO

Reply:
Ah - you don't need an AccessToken to call Checkout APIs? 💡  TIL

Reply:
Yes